doc/api/vm.md

```diff
@@ -171,6 +171,10 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
     to initialize the `import.meta`. This function has the signature `(meta,
     module)`, where `meta` is the `import.meta` object in the `Module`, and
     `module` is this `vm.SourceTextModule` object.
+  * `importModuleDynamically` {Function} Called during evaluation of this
+    module when `import()` is called.
```

Comment:
This gives me close to no information on how to use this option. I know when it's called and what information I'm given, but… what does it do? If I don't specify this, is `import()` going to work? Is there a default `importModuleDynamically`? What am I supposed to return?